app/extensions/safe/manageAuthNotifications.js

```diff
@@ -15,7 +18,61 @@ export const REQ_TYPES = {
     MDATA     : 'MDATA'
 };
 
-
+export const createAuthRequestElement = (authReqData) =>
+{
+  const R = React.createElement;
+  let reqType = 'authReq';
+  let reqTypeText = ' requests Auth Permission';
+  if (authReqData.contReq)
+  {
+    reqType = 'contReq'; 
+    reqTypeText = ' requests Container Access';
+  }
+
+  if (authReqData.mDataReq)
+  {
+    reqType = 'mDataReq'; 
+    reqTypeText = ' requests mData Access';
+  }
+
+  let children =
+  [
+    R('p', { className: 'blockText', key: authReqData[reqType].app.name }, `${authReqData[reqType].app.name} ${reqTypeText}`),
+    R('p', { className: 'blockText', key: authReqData[reqType].app.vendor }, `${authReqData[reqType].app.vendor}`),
+  ];
+
+  if ( authReqData[reqType].containers )
```

Comment:
I think we need to check for length here as this is still showing even if no container do afterwards (that or the containers are missing...)